ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/ConsumerGroupsTable.tsx

```diff
@@ -132,8 +189,12 @@ export function ConsumerGroupsTable({
               </Td>
             );
           case "topics":
-            const allTopics =
-              row.attributes.members?.flatMap((m) => m.assignments ?? []) ?? [];
+            const allTopics: { topicId: string; topicName: string }[] = [];
+
```

Comment:
The members should be considered along with the offsets. There's a chance all topics won't be present in one or the other.
```suggestion

            row.attributes.members?.flatMap((m) => m.assignments ?? []).forEach(a => {
              allTopics.push({ topicId: a.topicId, topicName: a.topicName });
            });
```